pkg/libvmi/storage.go

```diff
@@ -34,9 +34,14 @@ const (
 
 // WithContainerDisk specifies the disk name and the name of the container image to be used.
 func WithContainerDisk(diskName, imageName string) Option {
+	return WithContainerDiskAndPullPolicy(diskName, imageName, "")
+}
+
+// WithContainerDiskAndPullPolicy specifies the disk name, the name of the container image and Pull Policy to be used.
+func WithContainerDiskAndPullPolicy(diskName, imageName string, imagePullPolicy string) Option {
```

Comment:
```suggestion
func WithContainerDiskAndPullPolicy(diskName, imageName string, imagePullPolicy k8sv1.PullPolicy) Option {
```
And save a cast at line 44